Main.java

```diff
@@ -1,15 +1,32 @@
+import java.util.Scanner;
 /**
  *
- * @author 
+ * @author Isaac
  */
 public class Main {
 
-  /**
+  /** this program tells the user what order the numbers that they put in go first
    * The method that is executed when you hit the run button.
    * @param args the command line arguments
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    System.out.println ("Please enter a number.");
+    int first_number = input.nextInt();
+    System.out.println ("please enter a second number.");
+    int second_number = input.nextInt();
+
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. You have used snake_casing which is more of a Python thing.